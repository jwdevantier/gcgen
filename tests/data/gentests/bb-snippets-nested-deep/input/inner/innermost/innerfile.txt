
My inner file!

First foo:
# [[start foo
# end]]

Then bar:
# [[start bar
# end]]