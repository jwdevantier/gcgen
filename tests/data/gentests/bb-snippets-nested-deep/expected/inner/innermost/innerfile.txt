
My inner file!

First foo:
# <<? foo ?>>
foo from outer ctx!
scope["some-var"]: <<some-inner-val>>
/outer foo end
# <<? /foo ?>>

Then bar:
# <<? bar ?>>
bar from inner ctx!
scope["other-val"]: <<other-outer-val>>
# <<? /bar ?>>